docs/index.html

```diff
@@ -47,6 +47,7 @@
   <script src="//cdn.jsdelivr.net/npm/prismjs@1/components/prism-diff.min.js"></script>
   <script src="//cdn.jsdelivr.net/npm/prismjs@1/components/prism-docker.min.js"></script>
   <script src="//cdn.jsdelivr.net/npm/prismjs@1/components/prism-go.min.js"></script>
+  <script src="//cdn.jsdelivr.net/npm/prismjs@1/components/prism-http.min.js"></script>
```

Comment:
Seems unrelated- verify?

Reply:
It's from the html code block at `docs/development/heuristics/rest-api.md#L33` I would assume.

Reply:
Yea it's syntax highlighting for `http` code blocks

https://github.com/iver-wharf/iver-wharf.github.io/blob/88c3c3ac108e076bb41ebe357f6938136eb10e8f/docs/development/heuristics/rest-api.md#L33-L41